controllers/applicationvolumereplication_controller.go

```diff
@@ -444,6 +636,21 @@ func (r *ApplicationVolumeReplicationReconciler) generateManifest(obj interface{
 	return manifest, nil
 }
 
+func (r *ApplicationVolumeReplicationReconciler) newManifestWork(name string, homeClusterName string,
```

Comment:
(nit) Reword `homeClusterName` to `mcNamespace` or even `managedClusterName`? (home is more from the DR context, whereas for any arbitrary MW it is created on the MC namespace, hence the nit).